CHANGELOG.md

```
@@ -1,5 +1,32 @@
 # Changelog
 
+## [v6.0.0] - Unreleased
+
+### Breaking Changes
+
+* **Module path updated** from `github.com/apolloconfig/agollo/v5` to `github.com/apolloconfig/agollo/v6`:
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)